src/engine/strat_engine/backstore/crypt.rs

```diff
@@ -708,7 +708,7 @@ fn read_key(key_description: &str) -> Result<SafeMemHandle> {
             i => i,
         };
 
-        let mut key_buffer = SafeMemHandle::alloc(STRATIS_KEY_SIZE)?;
+        let mut key_buffer = SafeMemHandle::alloc(STRATIS_MEK_SIZE)?;
```

Comment:
This is not the MEK, it is the passphrase (what generates the KEK). I'd even hesitate to call it the KEK given that it is the passphrase from which the KEK is generated. The two are not the same.